docker/job-spec.json.s1-lar

```diff
@@ -0,0 +1,37 @@
+{
+  "command":"/home/ops/verdi/ops/slcp2pm/script/create_lar.sh",
+  "imported_worker_files": {
+    "$HOME/.netrc": "/home/ops/.netrc",
+    "$HOME/.aws": "/home/ops/.aws",
+    "$HOME/verdi/etc/settings.conf": "/home/ops/verdi/ops/slcp2pm/conf/settings.conf"
+
+  },
+  "recommended-queues" : [
+    "grfn-job_worker-large",
+    "aria-job_worker-large",
+    "mem-job_worker-large",
+    "dwr-job_worker-large",
+    "chevron-job_worker-small",
+    "floodmap-job_worker-small",
+    "urgent-response-job_worker-large"
+  ],
```

Comment:
I have updated the queues here to fit the Queues in JPL's ariamh:
https://github.com/aria-jpl/ariamh/blob/develop/docker/job-spec.json.s1-lar

^ Let me know if this is right

Reply:
@pymonger

Reply:
looks good @shitong01.